uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/dom/single/impl/BaseSingletonDOMElementFactory.java

```diff
@@ -72,6 +76,28 @@ public void execute() {
         });
     }
 
+    @Override
+    public E createDomElement(final GridLayer gridLayer,
+                              final GridWidget gridWidget,
+                              final GridBodyCellRenderContext context) {
+        widget = createWidget();
+        e = createDomElementInternal();
+
+        widget.addDomHandler(destroyOrFlushkeyDownHandler(context),
```

Comment:
@jomarko @manstis 
I think it could be better to move this _initialization_ code in a different method to keep it available even when the createDomElement method has been overridden